content/integrations/hipchat.md

```diff
@@ -28,7 +28,10 @@ Tick the checkbox if you want to be notified for every comment, in all configure
 
 You can now share graphs or send alerts to HipChat rooms using the syntax @hipchat-chat_name
 
-**NOTICE** : If your chat name contains special characters such as commas or brackets, they'll be escaped when sending notifications with the @ handle. You shouldn't have to worry about that, the autocomplete box automatically takes care of escaping chat names.
+**NOTICE** : 
+If you are using a hipchat API V1 token, if your chat name contains special characters such as commas or brackets, they'll be escaped when sending notifications with the @ handle. You shouldn't have to worry about that, the autocomplete box automatically takes care of escaping chat names.
```

Comment:
> If you are using a hipchat API V1 token, if your chat name contains...

If you are using a hipchat API V1 token and your chat handle contains...

> they'll be escaped when sending notifications with the @ handle. You shouldn't have to worry about that, the autocomplete box automatically takes care of escaping chat names.

What is the motivation for adding this? Are using trying to add, e.g. "@kent\,shultz" in the text box? If so, can we add an example, e.g. "if you username is "kent,shultz", do not enter "@kent\,shultz".

Also, we can just say once that the Datadog takes care of the escaping, e.g.

"If you are using a hipchat API V1 token and your chat handle contains special characters like commas or brackets, don't escape them when you enter the handle; the Hipchat integration does that for you."